cxx/emissions/string_alignment.cc

```diff
@@ -0,0 +1,77 @@
+#include <utility>
+#include "emissions/string_alignment.hh"
+
+void topn_alignments(int n, const std::string& s1, const std::string& s2,
```

Comment:
Slight preference to call this topk vs topn (since the former is more standard)

Reply:
Huh!  In the speech recognition and natural language processing worlds, this is always called "top n".  But a Google search suggests you are right.

Done.